src/env.ts

```diff
@@ -125,6 +125,11 @@ const envDefinitions = {
     def: "https://tron.coin.ledger.com",
     desc: "proxy url for trongrid API",
   },
+  API_HEDERA_MIRROR: {
+    def: "https://mainnet-public.mirrornode.hedera.com",
```

Comment:
This should be updated to use the unthrottled mirror node:

```suggestion
    def: "https://mainnet.mirrornode.hedera.com",
```

Reply:
Changing the mirror api url to `https://mainnet.mirrornode.hedera.com` makes the app stall indefinitely on the synchronizing step (where the url is queried to check for account associated with the ledger device). I changed it back to the previous url for the time being until this is addressed.